pkg/builtinchecks/yamls/unset-cpu-requirements.yaml

```
@@ -4,10 +4,10 @@ scope:
   objectKinds:
     - DeploymentLike
 remediation: >-
-  Set CPU requests and limits for your container based on its requirements.
+  Set CPU limits for your container based on its requirements.
```

As mentioned in the issue we need requests as it's required for scheduling and with no limits we allow bursts in CPU usage
- https://github.com/stackrox/kube-linter/issues/694
```suggestion
  Set CPU requests for your container based on its requirements.
```